qa/vagrant/build.gradle

```diff
@@ -64,7 +64,8 @@ String upgradeFromVersion
 
 String maybeTestsSeed = System.getProperty("tests.seed", null);
 if (maybeTestsSeed != null) {
-  seed = new BigInteger(maybeTestsSeed, 16).longValue()
+  def (masterSeed, testSeed) = maybeTestsSeed.tokenize(':')
+  seed = new BigInteger(masterSeed, masterSeed.size()).longValue()
```

Comment:
> Did not fail when I tried but I can change that anyway.

That doesn't make sense to me; I believe you but I need to test for my own understanding.